host-guest/symmetric-a-cyclodextrin_cyclopentanol/input/README.md

```diff
@@ -0,0 +1,3 @@
+# File Descriptions
```

Comment:
Maybe, um, a more detailed reference than `Mobley et al.` since there might be others I wrote that year. Or a link. Or just link to the GitHub repo. :)